Resources/config/orm.xml

```diff
@@ -95,7 +95,7 @@
 
         <service id="doctrine.orm.configuration" class="%doctrine.orm.configuration.class%" abstract="true" public="false" />
 
-        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" />
+        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" lazy="true" />
```

Comment:
Any idea how you'd do that? This is purely an implementation detail that doesn't leak through the interfaces. Given that next versions are going to need this "lazy", adding the dynamic check would only be a micro optim for soon to be legacy versions. Worth it?